<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadowSettings</class>
 <widget class="QWidget" name="ShadowSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="xOffsetLabel">
     <property name="text">
      <string>X offset:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="xOffsetBox">
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="yOffsetLabel">
     <property name="text">
      <string>Y offset:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="yOffsetBox">
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>33</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="blurLabel">
     <property name="text">
      <string>Blur radius:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="blurBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="opacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="opacityBox">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>70</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="removeButton">
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
